python/MooseDocs/extensions/MooseImageFile.py

```diff
@@ -0,0 +1,103 @@
+import re
+import os
+from markdown.util import etree
+import logging
+log = logging.getLogger(__name__)
+
+import MooseDocs
+from markdown.inlinepatterns import Pattern
+from MooseCommonExtension import MooseCommonExtension
+import utils
+
+class MooseImageFile(MooseCommonExtension, Pattern):
+    """
+    Markdown extension for handling images.
+    """
+
+    # Find !image /path/to/file attribute=setting
+    RE = r'!image\s+(.*?)\s+(.*)'
+    SETTINGS_RE = re.compile("([^\s=]+)=(.*?)(?=(?:\s[^\s=]+=|$))")
```

Comment:
Possibly... I didn't want to mess with MooseCarousel anymore than I had to. But, I was probably being to carful.
@brianmoose  what do you think?